client/src/containers/HomePage/HomePage.jsx

```diff
@@ -199,16 +200,30 @@ class HomePage extends Component {
       && !this.chartsRef.current.state.dialogOpen
     ) {
       // once you click "next" when the table is open, it will throw an error because the current target (reports) is technically closed
-      // the "step-after" part of the event can't mount because the target isn't found -> need to make it so that it manually skips over?
       this.setState({ stepIndex: index + 1 });
-    } else if (
+    } else if (index == 7 && [EVENTS.STEP_AFTER].includes(type)) {
+      const { current = {} } = this.mapRef;
+      const { leafletElement: map } = current;
+      map.flyTo(MAP_LOCATIONS.losAngelesCountyCenter, 6 /* ZOOM */, {
+        duration: 0.5,
+      });
+      setTimeout(() => { // Wait for map to pan + zoom to la county then move to next step so target can lock
+        this.setState({ stepIndex: index + (action === ACTIONS.PREV ? -1 : 1) });
+      }, 500);
```

Comment:
After the 'Unlock California' step, I clicked next (multiple times) and it displayed counties but was completely stuck on 'Unlock California'. Maybe we can wait for the user to zoom then check zoom state to be <= certain amount (or locktype to be counties)

Reply:
Also pressing back button on the Unlock step

Reply:
True, I think I should change the joyride to add a couple extra steps that make the user actually zoom in instead of doing it with "flyTo". Would better demonstrate how to zoom on the map and prevent issues from not being able to unlock California when it's not rendered anymore

Reply:
Marking this as resolved -> Added changes to make the user zoom in/out themselves after viewing county. Removes the need for manual zooming and the bugs that come with using it and breaking the map because counties weren't displayed yet